BookingProject/BookingWebAPI/BookingWebApi.Application/User/DTOs/UserChangeDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.User.DTOs
+{
+    public class UserChange
+    {
+        public string FieldName { get; set; } 
+        public string OldValue { get; set; }
+        public string NewValue { get; set; }
+    }
+    public record UserChangeDto(
+         string UserId, 
+         DateTime Timestamp ,
+         List<UserChange> Changes
+    );
```

Comment:
U can see the duplication. For this purpose create a project called Contracts and locate such contracts there. They will be shared among apps